### PR TITLE
Use local static files, tested with context and icon

### PR DESCRIPTION
**Pull request by FinalAngel**
This PR removes media referenced from other resources and makes implementation more reliable.

For testing purposes we cleaned up the following plugins:
- accordion
- accordion_item
- alert
- blockquote

which use the widgets:
- context
- icon

|before|after|
|---|---|
|image|image|

### CODE REVIEW

**Comment by vxsx** [comment]
currently picked context on the screenshot looks worse than before

File: aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+.iconpicker-popover .search-control {
+    width: 100% !important;
+}
+.iconpicker-popover table tfoot tr td {
+   padding-top: 5px;
+   border-top: 0;
+   background: none;
+}
+.iconpicker-popover .glyphicon- {
```
Comment (vxsx) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css]: something missing?

File: aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+(function ($) {
+    'use strict';
+
+    // shorthand for jQuery(document).ready();
+    $(function () {
+        /**
+         * Handles all reauired JavaScript for the aldryn-bootstrap3 addon.
+         *
+         * @class Bootstrap3
+         * @namespace CMS
```
Comment (vxsx) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js]: copypasta

File: aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+             */
+            contextWidget: function contextWidget() {
+                var data = $('.aldryn-bootstrap3-context').data();
+                var fieldName = data.context;
+                var contextInputs = $('.js-btn-group-context-' + fieldName + ' label input');
+                var selectedContextInput;
+
+                contextInputs.each(function (index, item) {
+                    var label = $(item).parent();
+                    var input = $(item);
```
Comment (vxsx) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js]: can we swap these two lines, and reuse `input` variable?

File: aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                        label.addClass('btn btn-' + item.value);
+                    } else {
+                        label.addClass('btn btn-default');
+                    }
+
+                    // set active states
+                    label.on('click', function () {
+                        var input = $(this).find('input');
+
+                        selectedContextInput.attr('checked', false);
```
Comment (vxsx) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js]: prop

File: aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+                        input.attr('checked', true);
```
Comment (vxsx) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js]: prop

File: aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css (hunk 1 above)
Comment (FinalAngel) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css]: no thats intentionally for the empty placeholder

**Pull request closed without merging**